### ISSUE TITLE
Unable to use default crd values

### ISSUE THREAD

**Issue by hardincm2** [zalando/postgres-operator#1461]
I'm trying to just override the config I want to customize when using the CRD-based configuration. However, this does not work because those default values are stored in a `values-crd.yaml` file and therefore will not get picked up when I helm install the postgres-operator. Is there a way around this? Or do I really have to copy the entire base configuration just to change a few values? 

Below is what I am trying to do:
```
# Postgres Operator configuration
# https://github.com/zalando/postgres-operator/blob/master/docs/reference/operator_parameters.md
postgres-operator:
  configTarget: "OperatorConfigurationCRD"

  configGeneral:
    docker_image: registry.opensource.zalan.do/acid/spilo-13:2.0-p6
  configKubernetes:
    enable_pod_antiaffinity: true
    secret_name_template: "{cluster}.{username}.credentials"
    storage_resize_mode: pvc
  configPostgresPodResources:
    default_cpu_limit: "2"
    default_cpu_request: 500m
    default_memory_limit: 1Gi
    default_memory_request: 250Mi
  configTeamsApi:
    enable_teams_api: false
```

And seeing this error because it is only picking up the configmap based values.
```
Error: UPGRADE FAILED: error validating "": error validating data: [ValidationError(OperatorConfiguration.configuration.aws_or_gcp.enable_ebs_gp3_migration): invalid type for do.zalan.acid.v1.OperatorConfiguration.configuration.aws_or_gcp.enable_ebs_gp3_migration: got "string", expected "boolean", ValidationError(OperatorConfiguration.configuration.connection_pooler.connection_pooler_max_db_connections): invalid type for do.zalan.acid.v1.OperatorConfiguration.configuration.connection_pooler.connection_pooler_max_db_connections: got "string", expected "integer", ...
```

Appreciate any help on this as this tool has been pretty awesome thus far!
- Thanks

**Comment by hardincm2** [comment]
I'm aware that I can explicitly provide the values file when trying to helm install, but ideally I'd like to just have the CRD values be the default when this helm dependency is installed to avoid custom logic for just this dependency in our CI pipelines. 

Is there any plan for the `values-crd.yaml` to just become the default in any of the future releases?

Thanks,
Cam

**Comment by FxKu** [comment]
We've now merged (#1224) the two values.yaml files. So there should be no more such errors regardless if you choose OperatorConfigurarion CRD or ConfigMap as your config target.
